### PR TITLE
[docs] add quick usage snippet to Whisper.

### PR DESCRIPTION
**Pull request by Vaibhavs10**
# What does this PR do?
The aim of this PR is to add a simple code snippet that anyone stumbling upon the Whisper documentation can look at and get to "Whispering" ASAP.

Internal [slack thread](https://huggingface.slack.com/archives/C04F8N7FQNL/p1717596323781449) for context

Once we agree on a structure I'd like to open a more bigger PR to add such for top visited text-generation models and image/ VLMs too.

## Before submitting
- [X] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#create-a-pull-request),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community can review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

@LysandreJik @stevhliu @osanseviero @pcuenca

### CODE REVIEW

File: docs/source/en/model_doc/whisper.md [hunk 1]
```diff
@@ -27,6 +27,23 @@ The abstract from the paper is the following:
...
 
+## Quick usage
+
+You can run Whisper in less than 4 lines of code and transcribe in less than a minute!
+
+```python
+import torch
+from transformers import pipeline
+
+whisper = pipeline("automatic-speech-recognition", "openai/whisper-large-v3", torch_dtype=torch.float16)
```
Comment (osanseviero) [inline docs/source/en/model_doc/whisper.md:38]: should we also load to GPU?

**Review by stevhliu — COMMENTED** [review]
I wonder if you could even put the code examples right above `# Overview` to help users get started even faster. They're probably not too interested in reading the abstract anyways.

You can probably even get away without adding any text to focus it even more on the code. The user would just see three tabs for selecting their GPU, copy/paste the corresponding code example, and start whispering.

For any other details (or if they're just curious), users could then _choose_ to explore the rest of the doc.

**Comment by osanseviero** [comment]
Not stale

**Comment by Vaibhavs10** [comment]
Not stale

**Review by LysandreJik — APPROVED** [review]
Let's merge it as is and eventually iterate on it later? Done is better than perfect

**Pull request merged**